src/render.js

```diff
@@ -108,20 +108,19 @@ const render = async function(
     globals,
     requestStats,
 ) {
-    const {url: entryPointUrl} = jsPackages[jsPackages.length - 1];
```

Comment:
There is no `url` property. This was aways setting `entryPointUrl` to `undefined`. Luckily for us, it was only used in logging.